platforms/access-bridge/src/errors.ts

```diff
@@ -0,0 +1,205 @@
+import { ServerResponse } from 'node:http';
+
+import { JWError, JWErrorResponse } from '@jwp/ott-common/types/errors.js';
+
+import { RequestMethod } from './http.js';
+
+/**
+ * List of possible error codes based on RFC-JW03-Delivery API
+ * https://www.notion.so/jwplayer/RFC-JW03-Delivery-API-fc35c5f5d300445eba4004de970d06b4
```

Comment:
Can you reference the public docs instead of notion? https://docs.jwplayer.com/platform/reference/error-codes

This is in the public repo, so partners and customers won't have access to our notion pages.